python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -122,13 +125,15 @@
     ("statusLWSCS", _LWSCS_STATUS_PERIOD),
     ("statusMonCS", _MONCS_STATUS_PERIOD),
     ("statusThCS", _THCS_STATUS_PERIOD),
+    ("check_all_commands_have_replies", _COMMANDS_REPLIED_PERIOD),
 }
 
 # The status methods and the intervals at which they are executed to be used
 # during commissioning. Not all can be used because the LabVIEW code of EIE
 # doesn't support all of them yet.
 METHODS_AND_INTERVALS_FOR_COMMISSIONING = {
     ("statusAMCS", _AMCS_STATUS_PERIOD),
+    ("check_all_commands_have_replies", _COMMANDS_REPLIED_PERIOD),
```

Comment:
Please consider eliminating `METHODS_AND_INTERVALS_FOR_COMMISSIONING` and adding one more item to the entries in ALL_METHODS_AND_INTERVALS: a boolean that indicates whether or not this command this is supported by the real dome. I see two advantages:
* You can easily enable new commands as the vendor supports them.
* It reduces duplication.

Reply:
I realized later (too late to want to log back in) that it might be nicer to have a constant that is a set or frozen set of unsupported commands. Then you can iterate over METHODS_AND_INTERVALS_FOR_COMMISSIONING while excluding anything in the unsupported command set (which will shrink to nothing over time as the vendor adds support for commands).

Still, adding a flag is a quite reasonable solution. Just passing it along in case you prefer it enough to redo it again.